src/anythingBut.ts

```diff
@@ -0,0 +1,4 @@
+import { simpleExp } from "./utils";
+
+const anythingBut = simpleExp<string>(exp => `[^${exp}]*`);
```

Comment:
This deviates a bit from the old API, but I feel renaming this function to `anyCharacterBut` making it output `[^${exp}]` would make the API more intuitive.